### PR TITLE
From release.df1505 ch isco- new mapping for avam, bfs, chisco

### PR DESCRIPTION
**Pull request by hamidsoleymani**

### CODE REVIEW

File: reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/model/OccupationLabelDto.java [hunk 1]
```diff
@@ -10,20 +9,24 @@
     private String language;
     private String classifier;
     private String label;
+    private final static String H = "-";
```
Comment (cjpsycos) [inline reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/model/OccupationLabelDto.java]: this should be called SEPARATOR

File: reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/strategy/OccupationLabelStrategy.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        final File file = csvFileReader.openFile(csvFileName);
+        final List<OccupationLabelDto> ocLableList = new ArrayList<>();
+        Map<String, OccupationLabelDto> map = new HashMap<>();
+        final Reader reader = Files.newBufferedReader(Paths.get(file.getPath()));
+        final CSVReader csvReader = new CSVReaderBuilder(reader).withSkipLines(1).build();//skip Header Row
+        String[] nextRecord;
+        StringBuilder stringBuilder = new StringBuilder();
+        while ((nextRecord = csvReader.readNext()) != null && nextRecord.length > 0) {
+            final OccupationLabelDto dto = new OccupationLabelDto();
+            //System.out.println(nextRecord[0]);
```
Comment (fabiangotzen) [inline reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/strategy/OccupationLabelStrategy.java]: Remove comment

File: reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/strategy/OccupationLabelStrategy.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            dto.setLanguage(nextRecord[3]);
+            dto.setClassifier(nextRecord[4]);
+            dto.setLabel(nextRecord[5]);
+            String id = stringBuilder.append(dto.getType()).append("-").append(dto.getCode()).append("-").append(dto.getLanguage()).append("-").append(dto.getClassifier()).toString();
+            dto.setId(id);
+            ocLableList.add(dto);
+            map.put(dto.getId(), dto);
+            stringBuilder.setLength(0);
+        }
+        if(ocLableList.size()!= map.values().size()){
```
Comment (fabiangotzen) [inline reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/strategy/OccupationLabelStrategy.java]: Can be removed or should throw an exception

File: reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/model/OccupationLabelDto.java (hunk 1 above)
Comment (hamidsoleymani) [reply reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/model/OccupationLabelDto.java]: Seperator or hyphen, so i choose the first letter of hyphen, so we dont have to inflate the line with SEPERATOR or HYOHEN.
Is that ok for you?

File: reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/strategy/OccupationLabelStrategy.java (hunk 3 above)
Comment (hamidsoleymani) [reply reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/strategy/OccupationLabelStrategy.java]: This is removed from in branch From_From_release. So it wont be there after merge

File: reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/model/OccupationLabelDto.java (hunk 1 above)
Comment (cjpsycos) [reply reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/model/OccupationLabelDto.java]: resolved in pr165

File: reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/strategy/OccupationLabelStrategy.java (hunk 2 above)
Comment (cjpsycos) [reply reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/strategy/OccupationLabelStrategy.java]: resolved in pr165

**Pull request merged**